File: base/src/main/java/com/fastaac/base/pagestate/ErrorCallback.java
package com.fastaac.base.pagestate;


import com.kingja.loadsir.callback.Callback;
import com.fastaac.base.R;


/**
 * author : wutao
 * e-mail : 670831931@qq.com
 * time   : 2019/05/17
 * desc   :
 * version: 1.0
 */
public class ErrorCallback extends Callback {
    @Override
    protected int onCreateView() {
        return R.layout.layout_error;
    }
}

File: base/src/main/java/com/aisier/architecture/pagestate/ErrorCallback.kt
package com.aisier.architecture.pagestate

import com.aisier.architecture.R
import com.kingja.loadsir.callback.Callback

/**
 * author : wutao
 * e-mail : 670831931@qq.com
 * time   : 2019/05/17
 * desc   :
 * version: 1.0
 */
class ErrorCallback : Callback() {
    override fun onCreateView(): Int {
        return R.layout.layout_error
    }
}